### ISSUE TITLE
Suggestions - After search more option, focus is lost

### ISSUE THREAD

**Issue by amyngu** [OfficeDev/office-ui-fabric-react#3389]
### Bug Report

- __Package version(s)__: office-ui-fabric-react 5.17.0
- __Browser and OS versions__: All

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) Yes

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable) Any site using 'search more' option with People Picker

#### Describe the issue:

Focus is lost when keying into search more option in suggestions

#### Actual behavior:
![focuslost](https://user-images.githubusercontent.com/31973030/32803010-fa2da3f0-c936-11e7-98a7-892e591803d1.gif)

After keying into search more option and hitting Enter, focus is lost (unable to use arrow keys to select)

#### Expected behavior:

After keying into search more options and hitting Enter, user is able to use arrow keys to select
